pkg/config/config.go

```diff
@@ -312,8 +312,8 @@ func InitConfig(config Config) {
 	// Forwarder
 	config.BindEnvAndSetDefault("additional_endpoints", map[string][]string{})
 	config.BindEnvAndSetDefault("forwarder_timeout", 20)
-	config.BindEnvAndSetDefault("forwarder_retry_queue_max_size", 0)
-	config.BindEnvAndSetDefault("forwarder_retry_queue_payloads_max_size", 15*megaByte)
+	_ = config.BindEnv("forwarder_retry_queue_max_size")
+	_ = config.BindEnv("forwarder_retry_queue_payloads_max_size")
```

Comment:
let's add comments to document that `forwarder_retry_queue_max_size` is deprecated, and that the default of `forwarder_retry_queue_payloads_max_size` is set in `pkg/forwarder`